assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -106,19 +102,22 @@ const WidgetAreaRenderer = ( { slug } ) => {
 	const widgetArea = useSelect( ( select ) => select( STORE_NAME ).getWidgetArea( slug ) );
 	const widgets = useSelect( ( select ) => select( STORE_NAME ).getWidgets( slug ) );
 
-	// State handled by WidgetRenderer instances, based on whether the widget
-	// renders content or `null`.
-	const [ activeWidgets, setActiveWidgets ] = useState( {} );
+	// Verify that widgets are active (do not render null)
+	const isActiveWidget = ( widget ) =>
+		widgets.some( ( item ) => item.slug === widget.slug ) &&
+		typeof widget.component === 'function' &&
```

Comment:
@caseymorrisus we don't need this check since we use `renderToStaticMarkup` now.

Reply:
@eugene-manuilov I think the check for a function here is just as relevant as before (used to throw if `widget.component` was not a function via `undefined is not a function`).

With the new changes, if `widget.component` were not a component, it would throw here with the error: `Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: <INVALID_ELEMENT_TYPE>` as `renderToStaticMarkup` expects a component as an argument. Checking the type prevents this error from being a possibility (can't find a typedef that makes me confident that `widget.component` is always defined/required, seems possible to be undef).

Reply:
Ok, good.